non-critical-infra/packages/encrypt-email-address/encrypt-email-address.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```

Reply:
I think this is based on suggestions of this pep: https://peps.python.org/pep-0394/

Reply:
Just to be sure no one ever tries to run the script with python2